src/validators/BaseValidator.ts

```diff
@@ -61,11 +63,37 @@ export abstract class BaseValidator<T> {
 	}
 
 	public parse(value: unknown): T {
+		// If validation is disabled (at the validator or global level), we only run the `handle` method, which will do some basic checks
+		// (like that the input is a string for a string validator)
+		if (this.isValidationEnabled === false) {
+			return this.handle(value).unwrap();
+		}
+
+		if (this.isValidationEnabled === null && !getGlobalValidationEnabled()) {
+			return this.handle(value).unwrap();
+		}
+
 		return this.constraints.reduce((v, constraint) => constraint.run(v).unwrap(), this.handle(value).unwrap());
 	}
 
+	/**
+	 * Sets if the validator should also run constraints or just do basic checks.
+	 * @param isValidationEnabled Whether this validator should be enabled or disabled. Set to `null` to go off of the global configuration.
+	 */
+	public setValidationEnabled(isValidationEnabled: boolean | null): this {
+		const clone = this.clone();
+		clone.isValidationEnabled = isValidationEnabled;
+		return clone;
+	}
+
+	public getValidationEnabled() {
+		return this.isValidationEnabled;
+	}
```

Comment:
Should be a getter, and account for global validation enabled. Then you can use this instead of two branches in every validator.

Reply:
Would be smart but this is supposed to represent the validators setting, not if the validator can run. That aside I could just make it a secondary getter